flow/include/flow/flow.h

```diff
@@ -100,6 +100,9 @@ extern StringRef strinc(StringRef const& str, Arena& arena);
 extern Standalone<StringRef> addVersionStampAtEnd(StringRef const& str);
 extern StringRef addVersionStampAtEnd(StringRef const& str, Arena& arena);
 
+// Return the number of combinations to choose k items out of n choices
+int nChooseK(int n, int k);
```

Comment:
You can squeeze more numbers by using unsigned?

Reply:
the caller uses int, so I chose int instead of unsigned or long.